src/api/api.js

```diff
@@ -64,6 +67,17 @@ class Api {
 
     this.socket.onclose = () => {
       console.log("[socket.close]");
+      EventEmitter.emit("setConnectStatus", false);
+
+      if (navigator.onLine && document.visibilityState === "visible") {
+        this.connect();
+      } else {
+        window.addEventListener("online", () => this.connect());
```

Comment:
if we become online, we also need to check if an app is in foreground and only then connect